exampleCourse/questions/element/dropdown/question.html

```diff
@@ -8,38 +8,47 @@
 </pl-question-panel>
 
 <p>
-  The 
-  <pl-dropdown answers-name="aristotle">
-    {{#params.aristotle}}
-        <pl-answer correct="{{tag}}">{{ans}}</pl-answer>
-    {{/params.aristotle}}
-  </pl-dropdown> 
-  is more than the sum of its parts.
+
+  <label>
```

Comment:
Why warp this in a label?

Reply:
https://html-validate.org/rules/input-missing-label.html

> The recommended way is using a <label> element either explicitly associated using the for attribute or by nesting the <input> element inside the <label>. For regular input fields the former is recommended and for checkboxes and radiobuttons the latter is recommended.

Reply:
How is this announced by screen readers?

Reply:
https://github.com/user-attachments/assets/a8998e5b-1630-49b4-be76-bd906cf6d55a

Reply:
It's missing some notion that the element is in-between the two label sections.

Reply:
Yep; that's exactly why I think this is a bad change.

Reply:
https://adrianroselli.com/2021/08/sentence-forms-not-mad-libs.html describes how to do this correctly

Reply:
Though they do that through multiple pieces of `aria-labeled-by`, which is an API that I wouldn't have access to if I remove it.

Reply:
An explicit `aria-label` attribute by way of a potential `label` attribute would meet that need too, you can just have `label="The blank is more than the sum of its parts."`.

Reply:
Changed in [785357d](https://github.com/PrairieLearn/PrairieLearn/pull/11844/commits/785357da395b470176b257d2ad9cac4482f141ef).